### ISSUE TITLE
Install `llvm-as` as `as` so that `-fno-integrated-as` continues working after GAS is removed?

### ISSUE THREAD

**Issue by DanAlbert** [android/ndk#1290]
I'm not sure if these are command line compatible. If they are we should install `llvm-as` as `as` so that `-fno-integrated-as` users get `llvm-as` instead of nothing after the GAS removal.

**Comment by pirama-arumuga-nainar** [comment]
[llvm-as](https://llvm.org/docs/CommandGuide/llvm-as.html) is not a GAS replacement.  It converts LLVM IR into LLVM bitcode.  AFAIK, there's no drop-in replacement for GNU assembler.

**Comment by DanAlbert** [comment]
Oh. Well then we really can't get rid of gas.

**Comment by DanAlbert** [comment]
Discussed further and while we can't make it a drop in replacement (and therefore won't work for this exact use case), the only broken use case is when `as` is invoked directly or when `-fno-integrated-as` is used. The former is migrated simply enough by using `clang` instead of `as` and prefixing assembler flags with `-Wa,`, and the latter is just removing that argument.

Closing this. Will fix the incorrect docs I just wrote :)
